### DISCUSSION TITLE
intermittent field_id not found error

### DISCUSSION THREAD

**Discussion by AxisNL** [codingjoe/django-select2#251] [Q&A]
Hi!

I get intermittent 'field_id not found' errors. On my local machine, doing a manage.py runserver, everything works fine. However, in my docker container it only works once every few requests.

If I start typing in my autocomplete field, I see ajax requests going to /select2/fields/auto.json?term=ange&field_id=IjVkOTJjOTE.. for example. If I just start typing something that is in the list, then most of the time it'll say 'the results could not be loaded', but every now and then upon typing, it'll output a response.

If I open up my browser's network inspect tool, I see most requests return a 404 'field_id not found', and some will return a 200 with the json. But all the requests are identical, except for the search term.

I don't understand why it's throwing these errors!

running python 3.11 and: 
Django==4.2.7
django-select2==8.1.2
gunicorn==21.2.0

These exact versions are the same in the docker image. 

On my dev machine I run manage.py runserver. In the docker image I have a simple nginx config with gunicorn running. No balancing, no caching components, no caching configuration in django (so the default LocMemCache is used).

![select2-errors](https://github.com/codingjoe/django-select2/assets/13782494/5b3e8998-9646-4615-8d34-1b6ad66d5dc5)

**Comment by CleitonDeLima** [comment]
How is your cache configured?

**Reply by AxisNL** [reply]
It isn't, so I assume the default LocMemCache is used.

**Reply by CleitonDeLima** [reply]
What is your cache timeout?

**Reply by AxisNL** [reply]
Nothing is configured, so I expect the default to be 300s (5 min)
